src/sheet/components/SheetDataEditorActions.tsx

```diff
@@ -168,7 +167,7 @@ export default function SheetDataEditorActions({
         clearable
         displayPlaceholderWhenSelected
         placeholder={t('sheet.filterByError')}
-        classes="min-w-48"
```

Comment:
Are you sure thats the correct fix for it? What if in some language the translation is longer? Cannot we try position the icon in a row somehow instead of using position absolute?

Reply:
This was fixed, right @emily-fields ?